src/specter/network/SpecterInterface.php

```diff
@@ -190,11 +190,8 @@ public function decodeAdditional(){
+			$pk->clientData["SkinId"] = base64_encode("Specter");
```

Comment:
that's incorrect. also, what in the world happened to the formatting?

Reply:
Not sure with the formatting. Just fixed it.  What's wrong with 193?

Reply:
it's not supposed to be base64'd. Only the skin data is.